docs/kqueen.rst

```diff
@@ -152,6 +152,142 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
+   prometheus:
+     image: prom/prometheus
+     restart: always
+     ports:
+       - 127.0.0.1:9090:9090
+     volumes:
+       - ./prod/prometheus/:/etc/prometheus/:Z
+       - /mnt/storage/kqueen/prometheus/:/prometheus/
+     links:
+       - api
+       - etcd
+
+2. Define  Prometheus scraper IP address in kqueen api service section:
+
+.. code-block:: yaml
+   KQUEEN_PROMETHEUS_WHITELIST: '172.16.238.0/24'
+
+There two ways to get metrics:
+
+- Kqueen API
+.. code-block:: bash
+   TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST <<kqueen_api_host>>:5000/api/v1/auth | jq -r '.access_token'); echo $TOKEN
+   curl -H "Authorization: Bearer $TOKEN" <<kqueen_api_host>>:5000/metrics/
+
+- Prometheus API
+Add scraper IP address to PROMETHEUS_WHITELIST configuration and then:
+
+.. code-block:: bash
+   curl <<prometheus_host>>:<<prometheus_port>>/metrics
+
+
+Provisioner configuration settings
+-------
+You can provision a Kubernetes cluster using various community of engines, such as:
+
+* Provision a Kubernetes cluster using the Google Kubernetes Engine:
+
+**Google Kubernetes Engine**
+1. Register account on GCE (https://console.cloud.google.com)
+2. Create your project
+3. Go to ```API’s & Services```  tab
+4. Go ```Credentials``` tab
+5. Click ```Create credentials```
+6. Choose ```Service Account key```
+7. Choose your Service account
+8. Choose Json format for key
+9. Download it
+10. On Kqueen UI choose ```Create Provisioner```
+11. Choose ```Google Kubernetes Engine```
+12. Insert your project ID (```Project info``` tab on the main page of GCE Dashboard https://console.cloud.google.com )
+13. Insert downloaded Json snippet, which contains key and submit provisioner creation
+14. On Kqueen UI choose ```Deploy Cluster```
+15. Choose defined GCE provisioner
+16. Specify cluster requirements
+17. Click ```Submit```
+18. Watch for cluster status from Kqueen main dashboard
+
+* Provision a Kubernetes cluster using the Azure Kubernetes Service:
+
+**Azure Kubernetes Service**
+1. Login into https://portal.azure.com with your azure account
+2. Follow official Microsoft guide https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application
+3. Get Application ID, Application Secret, Tenant ID (Directory ID), Subscription ID
+4. Pay attention to set ‘Owner’ role to your Application in Subscription settings to provide possibility of creating k8s clusters. (see latest steps in p.2). Save Application secret manually, because it will be unavailable from Azure Web UI after generating.
+5. Create Resource group from ‘Resource groups’ tab and get ‘Resource group name’
+6. Check that Application had ‘Owner’ role in Resource group. (go to ‘Resource groups’ -> your_group -> Access Control(IAM))
+7. Go to Kqueen ‘Create provisioner’ tab and choose AKS engine
+8. Set ‘Client ID’ as Application ID from p.3
```

Comment:
Set ‘Client ID’ as Application ID --> set application id as 'Client ID', since before in the instruction we deal only with app id

Reply:
fixed